wrapperclient.go

```diff
@@ -0,0 +1,53 @@
+package sfxlambda
+
+import (
+	"context"
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+	log "github.com/sirupsen/logrus"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+)
+
+var handlerFuncWrapperClient *sfxclient.HTTPSink
+
+func init() {
+	handlerFuncWrapperClient = sfxclient.NewHTTPSink()
+	if handlerFuncWrapperClient.AuthToken = os.Getenv("SIGNALFX_AUTH_TOKEN"); handlerFuncWrapperClient.AuthToken == "" {
+		log.Fatalf("Required environment variable SIGNALFX_AUTH_TOKEN is not set")
```

Comment:
This will cause the process to exit which will presumably cause the whole lambda to fail.  Do we really want their lambdas to all fail if they have misconfigured our wrapper?